infra/modules/vpc/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "project_name" {
+  type = string
+}
+
+variable "azs" {
```

Comment:
nit: could we spell out `availability_zones`? internally/local variables i care a bit less, but i'd like public interfaces to be as clear as possible and therefore want to minimize acronyms and abbreviations